pkgs/development/python-modules/pydruid/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pandas
+, prompt_toolkit
+, pycurl
+, pygments
+, pytest
+, pytestrunner
+, requests
+, six
+, sqlalchemy
+, tabulate
+, tornado
+}:
+
+buildPythonPackage rec {
+  pname = "pydruid";
+  version = "0.5.8";
+
+  src = fetchFromGitHub {
+    owner = "druid-io";
+    repo = pname;
+    rev = version;
+    sha256 = "0zb6zmklib26fzv5dvqzy0h0p1ljjgkklnjm66imc35mx5irpzcv";
+  };
+
+  patchPhase = ''
+    # timestamp is added to the end of the list, and dict keeps keys in insertion order
+    substituteInPlace tests/test_query.py --replace  \
+      "def expected_results_csv_reader():" "
+    EXPECTED_RESULTS_PANDAS = list(map(lambda x: dict(list(x.items())[1:] + [list(x.items())[0]]), EXPECTED_RESULTS_PANDAS))
+    def expected_results_csv_reader():"
+  '';
+ 
+  propagatedBuildInputs = [ pandas prompt_toolkit pycurl pygments requests six sqlalchemy tabulate tornado ];
```

really long, please list, please make into single item lines
```suggestion
  propagatedBuildInputs = [
    pandas 
    prompt_toolkit 
    ...
  ];
```

---

sure, is there a limit somewhere? (it fits to half screen for me so, below 120 characters - what is the standard max line width before wrapping?)

---

not really, but generally I do a max of ~3. Just helps with quickly digesting the expression